onnxruntime/server/grpc/grpc_app.cc

```diff
@@ -0,0 +1,29 @@
+#include "grpc_app.h"
+#include <grpcpp/health_check_service_interface.h>
+#include <grpcpp/ext/channelz_service_plugin.h>
+#include <grpcpp/ext/proto_server_reflection_plugin.h>
+namespace onnx_grpc = onnxruntime::server::grpc;
+
+namespace onnxruntime {
+namespace server {
+GRPCApp::GRPCApp(const std::shared_ptr<onnxruntime::server::ServerEnvironment>& env, std::string host, const unsigned short port) : m_service(env) {
+  m_server = std::unique_ptr<::grpc::Server>(nullptr);
+  ::grpc::EnableDefaultHealthCheckService(true);
+  ::grpc::channelz::experimental::InitChannelzService();
+  ::grpc::reflection::InitProtoReflectionServerBuilderPlugin();
+  ::grpc::ServerBuilder builder;
+  std::stringstream ss;
```

Comment:
I'd rather you just use strings to append.

Reply:
With what? sprintf or itoa?

Reply:
operator+

Reply:
error: no match for ‘operator+’ (operand types are ‘std::__cxx11::basic_string<char>’ and ‘const short unsigned int’) 

I'm going to stick with string stream for the whole thing.

Reply:
Up to you but you can just do `builder.AddListeningPort(host + ":" + std::to_string(port), ::grpc::InsecureServerCredentials());`

Reply:
oh cool - that's cleaner/what I was looking for earlier.